editProfile.php

```diff
@@ -1,5 +1,6 @@
 <?php
     // Author: Lauren Knight
+    //comment this for ya.
     // Description: Profile edit page
```

Comment:
(it wont let me add comments to specific lines for some reason so i have to do it like this)
maybe remove code that has been commented out that we aren't planning on using, like gender and contact method